js/index.js

```diff
@@ -40,3 +40,87 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+//add navbar text
+let navBar = document.querySelectorAll('a');
+// console.log(navBar);
+let navItems = siteContent["nav"];
+// console.log(navItems[`nav-item-1`])
+
+navBar.forEach(function(anchor,i) {
+  anchor.textContent = navItems[`nav-item-${i + 1}`];
+});
+
+//header section
+const headerText = document.querySelector('h1');
+headerText.textContent = siteContent['cta']['h1'];
+
+const headerBtn = document.querySelector('button');
+headerBtn.textContent = siteContent['cta']['button'];
+
+const headerImg = document.getElementById('cta-img');
+headerImg.src = siteContent['cta']['img-src'];
+
+
+//main section
+const mainSectionTopHeader = document.querySelectorAll('div.top-content div.text-content h4');
+mainSectionTopHeader[0].textContent = siteContent['main-content']['features-h4'];
+mainSectionTopHeader[1].textContent = siteContent['main-content']['about-h4'];
+
+const mainSectionTop = document.querySelectorAll('div.top-content div.text-content p');
+mainSectionTop[0].textContent = siteContent['main-content']['features-content'];
+mainSectionTop[1].textContent = siteContent['main-content']['about-content'];
+
+const middleImg = document.getElementById('middle-img');
+middleImg.src = siteContent['main-content']['middle-img-src'];
+
+const mainSectionBotHeader = document.querySelectorAll('div.bottom-content div.text-content h4');
+mainSectionBotHeader[0].textContent = siteContent['main-content']['services-h4'];
+mainSectionBotHeader[1].textContent = siteContent['main-content']['product-h4'];
+mainSectionBotHeader[2].textContent = siteContent['main-content']['vision-h4'];
+
+const mainSectionBot = document.querySelectorAll('div.bottom-content div.text-content p');
+mainSectionBot[0].textContent = siteContent['main-content']['services-content'];
+mainSectionBot[1].textContent = siteContent['main-content']['product-content'];
+mainSectionBot[2].textContent = siteContent['main-content']['vision-content'];
+
```

Comment:
this is just a little repetitive, think about how to keep DRY code (Dont Repeat Yourself) looks like youve got a lot of the same text around here, heres one way to make it more dynamic
```
const mainSectionBotHeader = document.querySelectorAll('div.bottom-content div.text-content h4');
const mainSectionBot = document.querySelectorAll('div.bottom-content div.text-content p');

for(let i = 0; i< mainSectionBot.length; i++){
  let section = '';
  switch(i){
    case 0: section = 'services'
    case 1: section = 'product'
    case 2: section = 'vision'
  }
  mainSectionBotHeader[i].textContent = siteContent['main-content'][`${section}-h4`];
  mainSectionBot[i].textContent = siteContent['main-content'][`${section}-content`];
}
```
with this all you need to add is a new case for the switch statment to add a new section, instead of repeating a whole new line of code again